packages/compass-indexes/src/components/create-index-form/columnstore-projection.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {
+  Editor,
+  EditorVariant,
+  CollapsibleFieldSet,
+} from '@mongodb-js/compass-components';
+
+type ColumnstoreProjection = {
+  useColumnstoreProjection: boolean;
+  toggleUseColumnstoreProjection: (useColumnstoreProjection: boolean) => any;
+  columnstoreProjection?: string;
+  columnstoreProjectionChanged: (columnstoreProjection: string) => any;
```

Comment:
Does `void` work here?
```suggestion
  toggleUseColumnstoreProjection: (useColumnstoreProjection: boolean) => void;
  columnstoreProjection?: string;
  columnstoreProjectionChanged: (columnstoreProjection: string) => void;
```

Reply:
Yes, thanks for pointing at it! I really didn't like these anys myself 🙈